api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -1582,11 +1582,10 @@ public void voidPatient_shouldReturnVoidedPatientWithGivenReason() throws Except
 	 * @see PatientService#voidPatient(Patient,String)
 	 */
 	@Test
-	@Ignore
-	// TODO fix: NullPointerException in RequiredDataAdvice
 	public void voidPatient_shouldReturnNullWhenPatientIsNull() throws Exception {
```

Comment:
Do you had the chance to take a look at this https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-ExceptionHandling?
We don't want to throws Exception because this could mask a travis failure .